### DISCUSSION TITLE
activate config portal by code

### DISCUSSION THREAD

**Discussion by StijnosYeah** [khoih-prog/WiFiManager_NINA_Lite#24] [Q&A]
IS their already an option forseen to reactivate the config portal by a call from software. As I read it now only reactivates if the stored credentials are not working or the configured wifi network is not available.
I would like to reactivate the config portal by holding down a button at powerup for example.

**Comment by khoih-prog** [comment]
The feature is already included by using **DRD** (Double-Reset Detector) or **MRD** (Multi-Reset Detector). 

Just read more carefully the section [**DRD detected and Config Portal is forcefully opened**](https://github.com/khoih-prog/WiFiManager_NINA_Lite#17-drd-detected-and-config-portal-is-forcefully-opened)
